features/device-config/rest/src/main/java/org/opennms/features/deviceconfig/rest/impl/DefaultDeviceConfigRestService.java

```diff
@@ -115,9 +123,10 @@ public ScheduleInfo(Date nextScheduledBackup, String scheduleInterval) {
         }
     }
 
-    public DefaultDeviceConfigRestService(DeviceConfigDao deviceConfigDao, DeviceConfigService deviceConfigService) {
+    public DefaultDeviceConfigRestService(DeviceConfigDao deviceConfigDao, DeviceConfigService deviceConfigService, SessionUtils sessionUtils) {
         this.deviceConfigDao = deviceConfigDao;
         this.deviceConfigService = deviceConfigService;
+        this.sessionUtils = sessionUtils;
```

Comment:
It's considered good style to use `Objects.requireNonNull` on such constructs. (Looks like I haven't reviewed the code above...)

Reply:
Objects.requireNonNull added